Dockerfile

```diff
@@ -24,7 +24,7 @@ RUN make raven_dev
 RUN make ostrich_dev
 
 # Install WPS
-RUN ["/bin/bash", "-c", "source activate wps && pip install -e ."]
+RUN ["/bin/bash", "-c", "source activate wps && python setup.py install"]
```

Comment:
Is there a reason why to evade using pip here?

Reply:
I was applying the cookiecutter, and this cookiecutter change was not from me: https://github.com/bird-house/cookiecutter-birdhouse/commit/317c4fb328a8a15b34957db2f5d5f2b639dab95e

Reply:
And the commit before that is https://github.com/bird-house/cookiecutter-birdhouse/commit/8deff6eebfd324e637f8a7301d073ce45fa1bf3f so using `pip install` never made it to the cookiecutter !

Reply:
I see some familiar names there. This is embarrassing. Oof.

Reply:
Oh my bad I was the one changing to use `pip install` and never backported to the cookiecutter https://github.com/Ouranosinc/raven/commit/1c1d342bd30923b4b9fc6d7b11399f412ad846b1  In my defense, I was new at that time and didn't know the cookiecutter existed.  Is `pip install` better than `python setup.py`?

Reply:
`pip install` makes it easier to grab any random missing dependencies from PyPI and in general is the promoted/accepted way to install modern python packages. It's also in the standard library and almost every python installation (unless you do a hyper minimal conda install). It's just better in general.

Reply:
Sure, wanna add it to cookiecutter?

Reply:
Sounds good. I'll tag you on the changes.

Reply:
I imagine the upstream cookiecutter changes are good to be merged here?

Reply:
yeah I'll refresh the cookiecutter again ... and again

Reply:
Each time I touch a new bird, I find something, backport to the cookiecutter, than refresh all the existing bird so ... I'll have more refresh to do anyways ...

Reply:
Thanks for the hard work!